From 9759c192e1c9f84808131a9b66adb480e2edfd80 Mon Sep 17 00:00:00 2001
From: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
Date: Mon, 16 Sep 2019 17:32:02 +0800
Subject: [PATCH 206/245] felix: add regmap address for TSN set

commit 42e59004b5be95dc063094cc8d6d26bc7c64fbe5 from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

QSYS_CIR_CFG is used in Qav set, and SYS_CNT is used in Qci sfi
status get. Reserved these two registers cause issue for TSN set.
modify these two registers to a right offset.

Signed-off-by: Xiaoliang Yang <xiaoliang.yang_1@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/net/ethernet/mscc/felix_regs.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/mscc/felix_regs.c b/drivers/net/ethernet/mscc/felix_regs.c
index b8bbd2992ed9..499ded27d06c 100644
--- a/drivers/net/ethernet/mscc/felix_regs.c
+++ b/drivers/net/ethernet/mscc/felix_regs.c
@@ -195,7 +195,7 @@ static const u32 felix_qsys_regmap[] = {
 	REG(QSYS_QMAXSDU_CFG_6,            0x00f62c),
 	REG(QSYS_QMAXSDU_CFG_7,            0x00f648),
 	REG(QSYS_PREEMPTION_CFG,           0x00f664),
-	REG_RESERVED(QSYS_CIR_CFG),
+	REG(QSYS_CIR_CFG,                  0x000000),
 	REG(QSYS_EIR_CFG,                  0x000004),
 	REG(QSYS_SE_CFG,                   0x000008),
 	REG(QSYS_SE_DWRR_CFG,              0x00000c),
@@ -288,7 +288,7 @@ static const u32 felix_sys_regmap[] = {
 	REG_RESERVED(SYS_MMGT_FAST),
 	REG_RESERVED(SYS_EVENTS_DIF),
 	REG_RESERVED(SYS_EVENTS_CORE),
-	REG_RESERVED(SYS_CNT),
+	REG(SYS_CNT,                       0x000000),
 	REG(SYS_PTP_STATUS,                0x000f14),
 	REG(SYS_PTP_TXSTAMP,               0x000f18),
 	REG(SYS_PTP_NXT,                   0x000f1c),
-- 
2.17.1

